python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -24,16 +24,18 @@ def get_partition_root(dataset):
     return file_path
 
 
-def get_folder_partition_root(folder):
+def get_folder_partition_root(folder, is_source=False):
     """Retrieve the partition root path using a dataiku.Folder.
 
     Args:
         dataset (dataiku.Dataset): Input or output dataset of the recipe used to retrieve the partition path pattern.
```

Comment:
There is a typo here
```suggestion
        folder (dataiku.Folder): Input or output folder of the recipe used to retrieve the partition path pattern.
```